### ISSUE TITLE
[BUG] Invalid CSS throws error in formbricks-js

### ISSUE THREAD

**Issue by mattinannt** [formbricks/formbricks#1132]
### Issue Summary

Implementing an invalid CSS selector seems to be throwing an error in formbricks-js (to be reconstructed).

![image](https://github.com/formbricks/formbricks/assets/675065/5b432eb2-521c-4fd5-ab4c-9d8506112906)

### Steps to Reproduce

1. (for example) Went to ...
2. Clicked on...
3. ...

### Expected behavior

In best case the css selector is already validated when it gets created. But also a wrong selector shouldn't throw an error in formbricks-js, but rather a console warning or error with the name of the action and a message that the selector is invalid.

### Environment

- [ ] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS:
- Node:
- npm:
```

**Comment by Ayush-Vish** [comment]
@mattinannt  Please assign me this issue under hacktober fest.

**Comment by jobenjada** [comment]
@Ayush-Vish its yours, go for it :)

**Comment by ShubhamPalriwala** [comment]
Hey @Ayush-Vish any update on this? Would like to know your progress or we might have to open it for others as well.

**Comment by Ayush-Vish** [comment]
@ShubhamPalriwala  Just give me 36 hours more ...

**Comment by Ayush-Vish** [comment]
@ShubhamPalriwala  You can now open the issue for others.
Thankyou for your pat

**Comment by pritish1906** [comment]
@mattinannt Please assign me this issue under hacktober fest.

**Comment by casanova1510** [comment]
hey @mattinannt @jobenjada @ShubhamPalriwala can you assign me this issue please? Will try my level best to resolve it ;)

**Comment by Ahsankhalid618** [comment]
let's do it.. assign me this issue @jobenjada

**Comment by ShubhamPalriwala** [comment]
Hey @pritish1906 since you commented first, will be assigning it to you for the next 24 hours so that you have the chance to open a PR for the same 🤞🏼

**Comment by shyamraghu** [comment]
Hi, @pritish1906 would you like to explain me how to reproduce this error, I can't able to understand. Thank you

**Comment by ortin779** [comment]
@ShubhamPalriwala  can you assign it to me? as it is already more than 24hrs for the previous assignee!! If that is okay

**Comment by ShubhamPalriwala** [comment]
Fair point! @ortin779 I'm assigning it to you now! It'd be super cool if you both (if want to) collaborate and ship this soon 🚀

**Comment by ortin779** [comment]
Sure, I had a discussion with @mattinannt , on what is the expected behavior. Will raise a pr Shortly

**Comment by ortin779** [comment]
@ShubhamPalriwala https://github.com/formbricks/formbricks/pull/1420 PR for the bug
